microk8s-resources/actions/enable.metrics-server.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Enabling Metrics-Server"
+
+"$SNAP/microk8s-enable.wrapper" rbac
+
+apiserver=$(wait_for_service apiserver)
+if [[ $apiserver == fail ]]
+then
+  echo "kube-apiserver did not start on time.  Proceeding."
+fi
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+use_manifest metrics-server apply
+
+refresh_opt_in_config "authentication-token-webhook" "true" kubelet
+refresh_opt_in_config "authorization-mode" "Webhook" kubelet
```

Comment:
Is webhook a requirement? I would expect it to work with the default AlwaysAllow

Reply:
not really required.  I was confused with the metrics-server documentation. :)